src/main/java/ch/uzh/ifi/hase/soprafs21/entities/SchedulingSession.java

```diff
@@ -21,15 +18,18 @@
     @OneToMany
     private List<Activity> activityList;
 
+
```

Comment:
I've tried to remove a couple of these empty lines, but there are too many to do it via Github 😅 
Please remove all added empty lines.

Reply:
The tests and the build works if scheduling session is commented, otherwise both fails because of scheduling session